webrender/src/render_backend.rs

```diff
@@ -214,6 +214,10 @@ impl RenderBackend {
                             }
                         }
                         ApiMsg::SetRootPipeline(pipeline_id) => {
+                            if scene.pipeline_map.get(&pipeline_id).is_none() {
```

Comment:
No this change is still wrong. We should be unconditionally calling set_root_pipeline_id. I originally did that but incorrectly changed the order.